internal/resources/fluentbit/resources.go

```diff
@@ -350,9 +350,6 @@ func MakeConfigMap(name types.NamespacedName, includeSections bool) *corev1.Conf
 
 `
 	fluentBitConfig = strings.Replace(fluentBitConfig, "{{ HTTP_PORT }}", strconv.Itoa(ports.HTTP), 1)
-	if includeSections {
-		fluentBitConfig = fluentBitConfig + "@INCLUDE dynamic/*.conf" + "\n"
```

Comment:
We still need `@INCLUDE dynamic/*.conf`. Add this to the `fluentBitConfig` string.